helm-charts/splunk-otel-collector/scripts/init-eks-fargate-cluster-receiver.sh

```diff
@@ -39,6 +39,9 @@ echo "Disabling k8s_cluster receiver for this instance"
 ./yq e 'del(.service.pipelines.metrics)' /conf/relay.yaml >/splunk-messages/config.yaml
 ./yq e -i 'del(.receivers.k8s_cluster)' /splunk-messages/config.yaml
 
+echo "Disabling prometheus kubernetes-apiserver receiver for this instance"
+./yq e -i 'del(.service.pipelines.metrics/histograms)' /splunk-messages/config.yaml
+
```

Comment:
Shall we delete/cleanup the `prometheus/kubernetes-apiserver` receiver and the exporter `signalfx/histograms` ?

Reply:
Tmk, since they are not used in any pipelines they won't be run by collector, so I kept it simple by removing just the pipeline

Reply:
right, it's just for clarity and avoid any confusion when user is reading the config. (I have seen this happening a lot in my SWAT days)

Reply:
sounds good, I'll remove them